lib/screens/settings_screen.dart

```diff
@@ -15,7 +18,31 @@ class SettingsScreen extends StatefulWidget {
 
 class _SettingsScreenState extends State<SettingsScreen> {
   User? get user => FirebaseAuth.instance.currentUser;
-  double rating = 0;
+  final InAppReview _inAppReview = InAppReview.instance;
+  Availability _availability = Availability.loading;
+
+  @override
+    void initState() {
```

Comment:
What is this extra level of indentation? Are you sure you are using the dart formatter?